custom/gsoc/note/api/src/main/java/org/apache/fineract/gsoc/note/api/NoteApiController.java

```diff
@@ -0,0 +1,114 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.fineract.gsoc.note.api;
+
+import static org.springframework.http.MediaType.APPLICATION_JSON_VALUE;
+import static org.springframework.http.MediaType.APPLICATION_PROBLEM_JSON_VALUE;
+
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.media.ArraySchema;
+import io.swagger.v3.oas.annotations.media.Content;
+import io.swagger.v3.oas.annotations.media.Schema;
+import io.swagger.v3.oas.annotations.responses.ApiResponse;
+import io.swagger.v3.oas.annotations.responses.ApiResponses;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import java.util.Collection;
+import lombok.RequiredArgsConstructor;
+import org.apache.fineract.portfolio.note.api.NotesApiResourceSwagger;
+import org.apache.fineract.portfolio.note.data.NoteData;
+import org.apache.fineract.portfolio.note.domain.NoteType;
+import org.apache.fineract.portfolio.note.exception.NoteResourceNotSupportedException;
+import org.apache.fineract.portfolio.note.service.NoteReadPlatformService;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping(value = "/gsoc/{resourceType}/{resourceId}/notes", consumes = { APPLICATION_JSON_VALUE }, produces = {
+        APPLICATION_JSON_VALUE, APPLICATION_PROBLEM_JSON_VALUE })
+@Tag(name = "Notes", description = "Notes API allows to enter notes for supported resources.")
+@RequiredArgsConstructor
+public class NoteApiController {
```

Comment:
I've run the project but couldn't access any end-points to test if they work or not.

I've tried to add `@Component` and remove it, but nothing changed.

Reply:
1.  are you trying the correct Docker image
2. look at my presentation on the last page; there is a curl statement that you can minimally modify and use here